e2e/cypress/integration/status/status_dnd_1_spec.js

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @status
+
+describe('DND Status - Setting Your Own DND Status', () => {
+    const dndTimes = [
+        'dndTime-30mins_menuitem',
+        'dndTime-1hour_menuitem',
+        'dndTime-2hours_menuitem',
+        'dndTime-Tomorrow_menuitem',
+        'dndTime-Custom_menuitem',
+    ];
+    const dndIconText = 'Do Not Disturb Icon';
+
+    before(() => {
+        // # Login as test user and visit channel
+        cy.apiInitSetup({loginAfter: true}).then(({team, channel}) => {
+            cy.visit(`/${team.name}/channels/${channel.name}`);
+        });
+    });
+
+    it('MM-8497_1 Set status DND with predefined end times', () => {
+        // # Loop through all predefined end times and verify them
+        for (let i = 0; i < 5; i++) {
```

Comment:
Doesn't this loop work for `i = 4` which would be the custom setting? I figured this test was just for the preset end times/